client/src/search.js

```diff
@@ -240,7 +245,11 @@ export class SearchWidget extends React.Component {
         });
 
         const results = indexResults.map(uri => {
-          return { title: this._map[uri], uri };
+          return {
+            title: this._map[uri].title,
+            uri,
+            popularity: this._map[uri].popularity
```

Comment:
This might not be needed/used but I think I put it in there when I was debugging the popularity in the results display.